browser.go

```diff
@@ -116,7 +116,10 @@ func CreateWindow(_, _ int, title string, monitor *Monitor, share *Window) (*Win
 			go w.framebufferSizeCallback(w, w.canvas.Width, w.canvas.Height)
 		}
 		if w.sizeCallback != nil {
-			go w.sizeCallback(w, w.canvas.GetBoundingClientRect().Width, w.canvas.GetBoundingClientRect().Height)
+			go w.sizeCallback(w,
+				int(w.canvas.GetBoundingClientRect().Width+0.5),  // Nearest non-negative int.
+				int(w.canvas.GetBoundingClientRect().Height+0.5), // Nearest non-negative int.
```

Comment:
I don't think rounding to nearest integer is needed here, it should just truncate:

```Go
int(w.canvas.GetBoundingClientRect().Width), int(w.canvas.GetBoundingClientRect().Height)
```